src/stores/user-store.js

```diff
@@ -52,24 +53,83 @@ class UserStore extends EventEmitter {
         }
     }
 
-    registerActions (action) {
-        if (action.ACTION_TYPE === ACTION_TYPE.addNewUser) {
-            const index = this._findUserIndexByName(action.user);
-            if (index === -1 || action.force) {
-                this._addNewUser(action.user);
-                this.emit(EVENT_TYPE.userAdded);
+    _findUser (user) {
+        return this._users.filter(anyUser => {
+            return (anyUser.name.toLowerCase().includes(user.name.toLowerCase()) &&
+                    anyUser.phone.toLowerCase().includes(user.phone.toLowerCase()) &&
+                    anyUser.email.toLowerCase().includes(user.email.toLowerCase()) &&
+                    anyUser.website.toLowerCase().includes(user.website.toLowerCase())
+            );
+        });
+    }
+
+    _defineSearchFields () {
+        return {
+            name:    this._searchedUser.name !== '',
```

Comment:
can be just 
```js
  name:    this._searchedUser.name !== '', -> !!this._searchedUser.name
```
or even better we can use `trim` method to crop space symbols at string's the begging and ending